Dockerfile-dev

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8-alpine
+
+RUN apk add build-base
```

Comment:
I think even this run command is not necessary.

Reply:
pip will fail without GCC, which alpine does not bring. It seems that Build-base the recommended way to get GCC.

https://wiki.alpinelinux.org/wiki/GCC